docs/development/codeguide_emacs.rst

```diff
@@ -1,207 +1,67 @@
-*********************************************
- Emacs setup for following coding guidelines
-*********************************************
+*******************************************
+Emacs setup for following coding guidelines
+*******************************************
 
-.. _flymake: http://www.emacswiki.org/emacs/FlyMake
```

Comment:
Do we need to retain a mention to use `flymake` instead of `flycheck` for Emacs 23 and older? Do people use Emacs that old?